go.mod

```diff
@@ -85,7 +85,7 @@ require (
 	github.com/rancher/norman v0.0.0-20200806175025-f974dbfb2734
 	github.com/rancher/rdns-server v0.0.0-20180802070304-bf662911db6a
 	github.com/rancher/remotedialer v0.2.6-0.20200401224317-04cc22157b83
-	github.com/rancher/rke v1.1.12
+	github.com/rancher/rke v1.1.13-rc1.0.20201222173119-6bdb7e95a572
```

Comment:
Can you update anything needed in apis/go.mod also, they look a bit stale: https://github.com/rancher/rancher/blob/master/pkg/apis/go.mod#L14

Reply:
will do